DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h

```diff
@@ -409,6 +415,11 @@ struct RecoContainer {
   auto getTPCITSTracksMCLabels() const { return getSpan<o2::MCCompLabel>(GTrackID::ITSTPC, MCLABELS); }
   auto getTPCITSTrackMCLabel(GTrackID id) const { return getObject<o2::MCCompLabel>(id, MCLABELS); }
 
+  // MFT-MCH
+  const o2::dataformats::GlobalFwdTrack& getGlobalFwdTrack(GTrackID gid) const { return getTrack<o2::dataformats::GlobalFwdTrack>(gid); }
```

Comment:
I wonder if/how the naming will change when the MID matching will be introduce (i.e. when we'll get MFT-MCH-MID tracks). Any thoughts on that ?

Reply:
FwdTrack is already taken by the base class, so the choice of global here was to be generic. The identification of GlobalMuonTrack to be performed by the user of `getGlobalFwdTrack` by reading a positive MIDmatchChi2, most notably to select the track-type at AOD production.